src/lib/util/anchor.ts

```diff
@@ -22,7 +20,9 @@ export function ensureAnchorHistory () {
 		// We want to allow links to absolute paths without
 		// using the history API. Also, if the target is not this frame we
 		// allow the default behavior (eg. if it should open in a new tab)
-		if (!href.startsWith(location.origin) || ($anchor.target !== "" && $anchor.target !== "_self")) {
+		if (!href.startsWith(location.origin) ||
+		   ($anchor.target !== "" && $anchor.target !== "_self") ||
+		   $anchor.dataset["routerSlot"] === "disable") {
```

Comment:
also open to name suggestions for the attribute and value.

Reply:
I really like the attribute name. I would prefer if we change the value `disable` to `disabled` to make it consistent with the naming of the `disabled` attribute.

Reply:
sounds good to me https://github.com/andreasbm/router-slot/pull/5/commits/d713a3053729607be67744fea2ac5e54ba0bfcba